examples/contracts/generics_forwarded/src/contract.rs

```diff
@@ -46,7 +48,7 @@ where
     QueryT: CustomMsg + DeserializeOwned + 'static,
     MigrateT: CustomMsg + DeserializeOwned + 'static,
     CustomMsgT: CustomMsg + DeserializeOwned + 'static,
-    CustomQueryT: CustomQuery + 'static,
+    CustomQueryT: CustomQuery + JsonSchema + 'static,
```

Comment:
Wondering if it should be a single trait like `sylvia::CustomQuery` aggregating those three for simpler usage.